src/java/apiview-java-processor/src/main/java/com/azure/tools/apiview/processor/Main.java

```diff
@@ -157,6 +158,15 @@ private static void processJavaSourcesJar(File inputFile, APIListing apiListing)
         apiListing.setLanguage("Java");
         apiListing.setMavenPom(reviewProperties.getMavenPom());
 
+        if(packageName.contains("spring")) {
```

Comment:
nit: let's use `groupId.contains("spring")` instead of `packageName`(which is groupId:artifactId) to ensure artifact names that happen to contain spring (not very likely) aren't classified as Spring libraries. As @JonathanGiles suggested below, we may want a different way to identify this but until then, we can go by the group name.